_includes/code/howto/java/src/test/java/io/weaviate/docs/model_providers/ConnectWeaviateEmbeddings.java

```diff
@@ -1,35 +1,32 @@
 package io.weaviate.docs.model_providers;
 
 // START-ANY
-
 import io.weaviate.client.Config;
 import io.weaviate.client.WeaviateAuthClient;
 import io.weaviate.client.WeaviateClient;
 import io.weaviate.client.base.Result;
 
+// END-ANY
 import static org.assertj.core.api.Assertions.assertThat;
 
 // START-ANY
-// Set these environment variables
-// WCD_HOSTNAME         Your Weaviate instance hostname
-// WCD_API_KEY          Your Weaviate instance API key
-
 public class ConnectWeaviateEmbeddings {
   public static void main(String[] args) throws Exception {
 
-    String host = System.getenv("WCD_W_EMB_HOSTNAME");
-    String apiKey = System.getenv("WCD_W_EMB_API_KEY");
+    // Best practice: store your credentials in environment variables
+    String wcdUrl = System.getenv("WEAVIATE_URL");      // Weaviate URL: "REST Endpoint" in Weaviate Cloud console
```

Comment:
I believe this should be "WEAVIATE_HOSTNAME" if it does not include the scheme (https://)